docs/glossary.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Glossary
+---
+
+## Terms
+
+### button
+#### See Also
+*   [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)
+
+### component
+A small, reusable piece of code that that returns an element to render.
+#### See Also
+*   [https://reactjs.org/docs/glossary.html#components](https://reactjs.org/docs/glossary.html#components)
+
+### computed property
+A property whose value is bound to that of one or more dependent properties.
+
+### Date
+#### See Also
+*   [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+
+### element
+Describes what you want to see on the screen, such as a heading DOM element with some text: `<h1>I'm an element!</h1>`.
+#### See Also
+*   [https://reactjs.org/docs/glossary.html#elements](https://reactjs.org/docs/glossary.html#elements)
+
+### event
+A message-passing strategy for indirect communication between components.
+#### See Also
+*   [enact/core/handle](../../modules/core/handle)
+
+### i18n
+A library of i18n routines written in JavaScript.
+#### See Also
+*   [enact/i18n](../../modules/i18n/$L)
+
+### JSX
+A JavaScript syntax extension that enables mixing JavaScript code with HTML/XML-like tags.
+#### See Also
+*   [Draft: JSX Specification](https://facebook.github.io/jsx/)
+
+### kind
+A kind is an object constructor created using the `@enact/core/kind` factory method.
+#### See Also
+*   [enact/core/kind](../../modules/core/kind)
+
+### layout
+The arrangement of controls on a page or view. The `@enact/ui/Layout` module provides a collection of components for laying out controls.
```

Comment:
Let's use 'components' here since we don't use the word 'controls' elsewhere (I think).